doc/source/user_guide/timeseries.rst

```diff
@@ -2149,12 +2149,9 @@ Time Zone Handling
 ------------------
 
 pandas provides rich support for working with timestamps in different time
-zones using the ``pytz`` and ``dateutil`` libraries.
+zones using the ``pytz`` and ``dateutil`` libraries or ``datetime.timezone``
```

Comment:
I think that :class:`datetime.timezone` will work.

Reply:
Looks like there are a few more mentions of what's supported

- Reword L272 to include datetime.timezone
- Add a daetetime.timezone example in the example ~L2181